package.json

```
@@ -34,6 +34,7 @@
     "format": "prettier . --write",
     "type-check": "tsc --build",
     "changeset": "changeset",
+    "perversion-packages": "turbo run perversion",
```

```suggestion
    "perversion-packages": "turbo run perversion",
```

This is a very perverted change.

---

Oooof good catch 😅 fixed!